apps/prairielearn/src/pages/partials/title.ejs

```diff
@@ -55,7 +55,7 @@
             courseName += ', ' + course_instance.short_name;
         }
         navTrace.push(courseName);
-    } else if (typeof navPage !== 'undefined' && !navPageIgnore.includes(navPage)) {
+    } else if (typeof navPage !== 'undefined' && !navPageIgnore.includes(navPage) && !displayedTitle) {
```

Comment:
The change in `news_items` to include "News" in the title caused the page to become "News - News" if the page is opened outside of a course context. This is to avoid that problem.

Reply:
👍 our title system is horrible, I can't easily reason through if this would affect other pages. It's probably fine?